server/settings/publish_plugins.py

```diff
@@ -642,6 +651,13 @@ class ExtractReviewOutputDefModel(BaseSettingsModel):
         default_factory=ExtractReviewLetterBox,
         title="Letter Box"
     )
+    fill_missing_frames: str = SettingsField(
+        title="Handle missing frames",
+        default="closest_existing",
+        description="How to handle frames that are missing from entity frame "
```

Comment:
Does it really have to do with `missing entity frame range`?

I feel it's more of:
> "How to handle gaps in sequence frame ranges."